library/pod-security-policy/users/template.yaml

```diff
@@ -102,8 +102,15 @@ spec:
 
         get_user_violation(params, container) = msg {
           not get_field_value("runAsUser", container, input.review)
-          params.rule != "RunAsAny"
-          msg := sprintf("Container %v is attempting to run without a required securityContext/runAsUser. Allowed runAsUser: %v", [container.name, params])
+          params.rule = "MustRunAs"
+          msg := sprintf("Container %v is attempting to run without a required securityContext/runAsUser", [container.name])
```

Comment:
I see I left the param field out. The formatting was a bit off, but very useful.